assets/css/tokens.css

```
@@ -0,0 +1,41 @@
+/* ABOUTME: Design-token palette (6 semantic colors × light/dark) plus font/size/radius tokens for the Design 4 restyle. */
+/* ABOUTME: :root is the default "paper" palette; [data-mode] is the manual override; themes.css layers per-theme values on top. */
+
+:root {
+    --bg: #ffffff;
+    --bg2: #f7f5f1;
+    --ink: #191817;
+    --mut: #8c877e;
+    --line: #ece9e3;
+    --accent: #b8532c;
+    color-scheme: light;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)